### ISSUE TITLE
Include Instantiate2 message

### ISSUE THREAD

**Issue by mikedotexe** [CosmWasm/cw-multi-test#39]
There's a very interesting message, `Instantiate2` that was introduced.

https://github.com/CosmWasm/wasmd/blob/5537735e00a3f96893a27294a992a65415e49dce/x/wasm/keeper/contract_keeper.go#L71

This is super handy and you can see it in action in Simon's virus contract:
https://github.com/CosmWasm/cosmwasm/tree/v1.2.1/contracts/virus

When I was trying to write tests I got:

```
    1: Unsupported wasm message: Instantiate2 { admin: Some("juno18smcd7hmzj4pt04vmt693u6lcvtxr44e20kz3w"), code_id: 1, label: "Confirming the first instantiation", msg: {}, funds: [], salt: Binary(5b39312c35302c35322c35372c34342c35302c35322c35335d) })
test tests::core_tests::call_create_simple ... ok
```

I'm not super familiar with this repo to dive in and help, but wanted to create this issue. Perhaps someday I can contribute

**Comment by 0xFable** [comment]
Also have ran into this. 
Looking for both CodeIdData and Instantiate2 support. Looking into seeing if I can do it

**Comment by whalelephant** [comment]
hey! this is going to to be helpful for us too, let me know @0xFable @mikedotexe if you guys have made progress / need help?

**Comment by DariuszDepta** [comment]
See also: https://github.com/CosmWasm/cosmwasm/issues/1648

**Comment by DariuszDepta** [comment]
Handling `Instantiate2` is implemented on main PR #65
